modules/howtos/pages/managing-connections.adoc

```diff
@@ -42,6 +42,9 @@ The client fetches the full address list from the first node it is able to conta
 [#connection-strings]
 == Connection Strings
 
+WARNING: the backend implementation of connection strings parameters changed substantially in 4.0 and is not currently fully documented. This will be resolved in a future 4.x release.
```

Comment:
```suggestion
WARNING: The backend implementation of connection strings parameters changed substantially in 4.0 and is not currently fully documented.
This will be resolved in a future 4.x release.
```